app/src/controllers/appInfo/selectors.js

```diff
@@ -57,6 +57,8 @@ export const analyticsEnabledSelector = (state) =>
   extensionsConfigSelector(state)[ANALYTICS_ALL_KEY] === 'true';
 export const analyzerExtensionsSelector = (state) => extensionsSelector(state).analyzers || [];
 export const authExtensionsSelector = (state) => uatInfoSelector(state).authExtensions || {};
+export const ssoUsersOnlySelector = (state) =>
+  extensionsConfigSelector(state)['server.users.sso'] === 'true';
```

Comment:
Pls move the `'server.users.sso'` value to the constant as it is done for other similar properties.